docs/content/components/button.mdx

```diff
@@ -7,9 +7,22 @@ import { Box, ActionGroup, Button, Stack } from '@marigold/components';
 
 # Button
 
+## Demo
+
+<MarigoldTheme>
+  <ActionGroup space="small">
+    <Button>Primary</Button>
+    <Button variant="secondary">Secondary</Button>
+    <Button variant="ghost">Ghost</Button>
+    <Button variant="text">Text Only</Button>
+  </ActionGroup>
+</MarigoldTheme>
+
 ## Description
 
-With the Button component you can render a `button` element.
+The Button component renders a `<button>` element.
+Buttons allow users to perform an action or to navigate to another page.
```

Comment:
Never use a button for navigation! That's what links are for!

Reply:
copy paste from react-spectrum 😀